app/src/main/java/com/sdp/movemeet/utility/PermissionChecker.java

```diff
@@ -0,0 +1,57 @@
+package com.sdp.movemeet.utility;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.util.Log;
+
+import androidx.core.app.ActivityCompat;
+
+public abstract class PermissionChecker {
+
+    private static String TAG = "permission checker tag";
+
+    /**
+     * Checks that permission for accessing GPS location is granted
+     * @return true if permission is granted, false otherwise
+     */
+    public static boolean isLocationPermissionGranted(Activity activity) {
+        return (ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED ||
+                ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_COARSE_LOCATION) == PackageManager.PERMISSION_GRANTED);
+    }
+
+    public static boolean isStorageWritePermissionGranted(Activity activity) {
+        if (Build.VERSION.SDK_INT >= 23) {
```

Comment:
Does it really mean that below Android SDK version 23, the user doesn't have to grant access to his storage (for writing and reading files) and this permission is granted by default? ^^'